plugins/inputs/prometheus/sample.conf

```diff
@@ -109,8 +109,11 @@
   # headers = {"X-Special-Header" = "Special-Value"}
 
   ## Specify timeout duration for slower prometheus clients (default is 3s)
+  # timeout = "3s"
+  
+  ##   deprecated in 1.26; use the timeout option
   # response_timeout = "3s"
-
+  
```

Comment:
I would say remove the option in the `sample.conf` to have the example show the recommended way of configuration but _keep_ the option in code. @powersj  what do you think?